test/integration/targets/connection_aws_ssm/runme.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -eux
+
+CMD_ARGS=$*
+
+# Destroy Environment
+cleanup() {
+
+    cd ../connection_aws_ssm
+
+    # As mentioned at last exception on link
+    # https://github.com/koalaman/shellcheck/wiki/Sc2086
+    # disabling shellcheck
+    # shellcheck disable=SC2086
+    ansible-playbook -c local aws_ssm_integration_test_teardown.yml $CMD_ARGS
```

Comment:
Use the script args as an array and the shellcheck disable won't be needed.

```suggestion
    ansible-playbook -c local aws_ssm_integration_test_teardown.yml "${CMD_ARGS[@]}"
```

Reply:
Hi @mattclay ,
Fixed this issue in the commit https://github.com/ansible/ansible/pull/49652/commits/1bd4a9f2a38485603129cda99ab7324026ef9b7d